### DISCUSSION TITLE
Filter invalid message in QueryNode

### DISCUSSION THREAD

**Discussion by RongHouchao** [milvus-io/milvus#20250] [Q&A and General discussion]
filter invalid insert message, segments are excluded segments
这个错误如何排查，这是什么意思

**Comment by RongHouchao** [comment]
![image](https://user-images.githubusercontent.com/117245915/199383901-5851c16c-f604-4082-8872-065499c354ae.png)

**Reply by yhmo** [reply]
只是一个debug日志，是过滤掉某些特例用的

**Reply by RongHouchao** [reply]
哦哦，多谢
